### PR TITLE
WIP read and convert files

### LINKED ISSUE #3: Read and convert files
- [] Create a function which reads the DKB example .csv file and creates an output file with the same data in the [YNAB format](https://docs.youneedabudget.com/article/921-importing-a-csv-file)
- [] Add [tape](https://www.npmjs.com/package/tape) as a test runner
- [] Write a test for the created function which is run by tape in the npm script `test`

### PR DESCRIPTION
**Pull request by Levino**
Fixes #3 

Work in progress.

### CODE REVIEW

**Comment by Levino** [comment]
I created this PR so we can conveniently discuss your current code and attempt at the solution.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      Outflow: outflow,
+      Inflow: inflow,
+    })
+  }
+  // inlcude map
+  // include mapper function
+  // define mapper in global
+  // test mapper in edge cases
+
+  inputJson.map(createYNABJsonMapper)
```
Comment (Levino) [inline index.js]: Nice. You iterate through the `inputJson` Array and create a new Array with the mapped values. And then you do nothing with this new Array but forget about it and return the old one in the next line.
```suggestion
  return inputJson.map(createYNABJsonMapper)
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  return outputJson
```
Comment (Levino) [inline index.js]:
```suggestion
```

File: dkb_sample_line_YNAB.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Date,Payee,Memo,Outflow,Inflow
+10/02/18,Hans Foo,Semestergebuehren,50,
+10/03/18,Foo Hans ,Semestergebuehren,,10
```
Comment (Levino) [inline dkb_sample_line_YNAB.csv]: Is this emptyspace on purpose?
```suggestion
10/03/18,Foo Hans,Semestergebuehren,,10
```

Comment (Levino) [reply dkb_sample_line_YNAB.csv]: Where is this file bing used? I guess it is not. Remove it please.
